en/micro-integrator/docs/references/connectors/redis-connector/redis-connector-configuration.md

```diff
@@ -0,0 +1,36 @@
+# Setting up the Environment 
+
+The Redis connector allows you to access the Redis commands through the WSO2 EI. Redis stands for remote dictionary server. Redis store /serve that stores data as key-value pairs and this key-value store can be used as a database.
+
+## Setting up the environment
+
+Before you start configuring the Redis connector, you also need WSO2 MI, and we refer to that location as <PRODUCT_HOME>.
+
+To configure the SMPP connector, copy the following client libraries from the given locations to the `<PRODUCT_HOME>/lib` directory.
+
+* [ jedis-2.1.0.jar](https://mvnrepository.com/artifact/redis.clients/jedis/2.1.0)
+
+## Setting up the Redis server 
+
+1. Navigate to the [Redis server](http://redis.io/download) and download it. Please follow the steps given in this page to install the Redis server in your local machine.
+2. After setting up the **Redis Server** navigate to Redis installed location and execute **src/redis-server** command.
+3. In the terminal you will see the Redis **port** and **PID** as shown bellow.
+    
+   <img src="../../../../assets/img/connectors/redis-server.png" title="Redis server" width="600" alt="Redis server"/> 
+ 
+4. You can interact with Redis using the built-in client. For this please open new terminal and navigate to Redis installed location. Then enter **src/redis-cli** command.
```

Comment:
IMO there is no need to say - please. 
open `a` new terminal

Reply:
Same with this command as well. Shall we present it properly?

Reply:
Updated.